sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts

```diff
@@ -511,8 +511,8 @@ export class EducationProgramOfferingInstitutionsController extends BaseControll
     @Query("validation-only", new DefaultValuePipe(false), ParseBoolPipe)
     validationOnly: boolean,
   ): Promise<PrimaryIdentifierAPIOutDTO[]> {
-    // Read the entire file content.
-    const fileContent = file.buffer.toString();
+    // Read the entire file content in ASCII encoding.
+    const fileContent = file.buffer.toString("ascii");
```

Comment:
I am ok to have the constant right now but it may not be suitable for all the cases because we are passing this value to different libraries. For instance, if this change in the future to UTF-8 some libraries can define the values as `UTF-8` and others may be `utf8`, which would require a case-by-case analysis. The bottom line, this config as it is defined an consumed may not be achieving the point of changing its value and having the system behave as expected.